appinventor/buildserver/src/com/google/appinventor/buildserver/Project.java

```diff
@@ -250,8 +254,37 @@ public void setVName(String vname) {
    * @return useslocation property
    */
   public String getUsesLocation() {
```

Comment:
Rather than having these methods return `String`, maybe it makes sense for them to return `boolean` instead? It seems like all we end up doing is checking whether they equal "False" anyway so it would make the code more readable.